ts/screens/wallet/FirstTransactionSummaryScreen.tsx

```diff
@@ -0,0 +1,245 @@
+  }
+
+  /**
+   * Depending on the comparison between the amount on the notice and the amount saved remotely by the lender
+   * it will be displayed a different component. If the values differ, then the user can display both the value
+   * and a brief exmplanation.
+   */
+  private isAmountUpdated() {
+    if (
+      transactionDetails.currentAmount.match(
+        transactionDetails.notifiedAmount
+      ) === null
+    ) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+
+  private getSummary() {
+    if (this.isAmountUpdated() === true) {
+      return (
+        // if the amount had been updated, it will be displayed and presented with a brief description
+        <UpdatedPaymentSummaryComponent
+          navigation={this.props.navigation}
+          amount={`${transactionDetails.notifiedAmount}`}
+          updatedAmount={`${transactionDetails.currentAmount}`}
+          expireDate={`${transactionDetails.expireDate.toLocaleDateString()}`}
+          tranche={`${transactionDetails.tranche}`}
+        />
+      );
+    } else {
+      return (
+        <PaymentSummaryComponent
+          navigation={this.props.navigation}
+          amount={`${transactionDetails.currentAmount}`}
+          expireDate={`${transactionDetails.expireDate.toLocaleDateString()}`}
+          tranche={`${transactionDetails.tranche}`}
+        />
+      );
+    }
+  }
+
+  public render(): React.ReactNode {
+    return (
+      <Container>
+        <AppHeader>
+          <Left>
+            <Button transparent={true} onPress={() => this.goBack()}>
+              <Icon name="chevron-left" />
+            </Button>
+          </Left>
+          <Body>
+            <Text>{I18n.t("wallet.firstTransactionSummary.header")}</Text>
+          </Body>
+        </AppHeader>
+
+        <Content original={true}>
+          <Grid style={[styles.padded, WalletStyles.backContent]}>
+            <Row>
+              <Col size={5}>
+                <View spacer={true} large={true} />
+                <H3 style={WalletStyles.white}>
+                  {" "}
+                  {I18n.t("wallet.firstTransactionSummary.title")}
+                </H3>
+                <H1 style={WalletStyles.white}>{`${
+                  transactionDetails.paymentReason
+                }`}</H1>
+              </Col>
+              <Col size={1}>
+                <View spacer={true} large={true} />
+                <Image
+                  source={require("../../../img/wallet/icon-avviso-pagopa.png")}
+                />
+              </Col>
+            </Row>
+            <View spacer={true} large={true} />
+          </Grid>
+
+          {this.getSummary()}
+
+          <View spacer={true} large={true} />
+          <Grid style={styles.padded}>
+            <Row>
+              <Text bold={true}>
+                {I18n.t("wallet.firstTransactionSummary.entity")}
+              </Text>
+            </Row>
+            <Row>
+              <Text>{`${entityDetails.name}`}</Text>
```

Comment:
why do we need the `${entityDetails.name}` notation here ? cannot we just use `{entityDetails.name}` ?